sections/appendix/author_category.tex

```diff
@@ -1,12 +1,18 @@
 \subsection{The author-category model}\label{sec:combination}
-\vejleder[inline]{hvorfor og formål?}
 We have created a combination model, as an extension of our metadata models, to see whether using multiple metadata fields at the same time to draw topics, would affect the performance of the topic model.
 The idea is that this model combination should give insight into what a model learns when multiple metadata influence the topics chosen.
 The model we have created is the Author-Category combination model.
 As the name suggests, this model includes an author-topic distribution and a category-topic distribution, and the plate notation can be seen in \autoref{fig:author_category_lda}.
 
-To combine the Author and Category metadata, we multiply the topic distributions $\theta_a$ and $\theta_c$ when sampling a new word topic.
-Though, before sampling the word topic, we normalize the product of the multiplication to sum to 1, which also makes it a distribution. \vejleder[inline]{mathify this}
+To combine the Author and Category metadata, we multiply the topic distributions $\theta_a$ and $\theta_c$ and the topic word $\beta_w$ when sampling a new topic $t$ for word $w$.
+\begin{equation}
+	p_z = \frac{\theta_a}{C(\theta_a)} \cdot \frac{\theta_c}{C(\theta_c)} \cdot \frac{\beta_w}{C(\beta_w)}
+\end{equation}
+Where C() is a function that counts number of times the word or document is used within a topic.
+Though, before sampling a new topic, we normalize it by the $L1$ norm, where we divide by sum of all variables, which also makes it a distribution. 
```

Comment:
```suggestion
Though, before sampling a new word topic, we normalize $p_z$ by the $L1$ norm where we divide by the sum of all variables, which also makes it a distribution. 
```
Hvad 'it' er, er ikke tydeligt. Hvis det ikke er p_z skal det i hvert fald tydeliggøres.